src/transform/FooterLegal.css

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Some funky spacing here :] Extra newline at top and at the start and finish of the rule body for `.pagelib_footer_legal_contents`.